packages/design-system/src/components/ChoiceList/Choice.test.jsx

```diff
@@ -113,7 +113,6 @@ describe('Choice', () => {
     const input = wrapper.find('input');
 
     expect(input.hasClass('ds-c-choice')).toBe(true);
-    expect(input.hasClass('ds-c-choice--right')).toBe(false);
```

Comment:
Removed this because we no longer support the right to left alignment of choice components and I think that should have been removed previously.

https://github.com/CMSgov/design-system/commit/680304ee58f908d1791842e7bd5e5d92076fae88
https://github.com/CMSgov/design-system/commit/f49673b5b718c9fc1395d1cb3f5ee3f6d441b173